### PR TITLE
Remove "duplicate" image-all target (use images-all instead)

### PR DESCRIPTION
**Pull request by Brian-McM**
## Description

This target doesn't seem to build the arm64 image, it just hangs. The might be some oddness with the sub-image-% target being redeclared that's causing this, but all I know is that images-all works and it's used in the CI but image-all doesn't and it's only used for the release building.

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Review by tmjd — APPROVED** [review]
LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Pull request merged**
